### DISCUSSION TITLE
UI Change

### DISCUSSION THREAD

**Discussion by arduinoturkbatu** [community/community#62200] [Mobile]
Seeing a Material 3 (You) design change on mobile app sound cool for the app design.

**Comment by mxie** [comment]
Hi @arduinoturkbatu - thanks for your feedback! Can you tell us more about what from the new Material design you'd like to see in terms of the GitHub Mobile app?

**Reply by arduinoturkbatu** [reply]
Hi @mxie ,
I think Material UI 3's bottom navigation bar's design and its animations are very good. Also, Material 3 introduced a large sized top nav bar and I would like to see a usage in the Settings tab or other tabs of the app.

I linked the relating pages on M3 website:

[https://m3.material.io/components/navigation-bar/overview](https://m3.material.io/components/navigation-bar/overview) (Navigation bar)
[https://m3.material.io/components/top-app-bar/guidelines](https://m3.material.io/components/top-app-bar/guidelines) (Top app bar)
